math/src/elliptic_curve/short_weierstrass/curves/bls12_377/curve.rs

```diff
@@ -96,21 +110,21 @@ mod tests {
 
     #[test]
     #[should_panic]
-    fn create_invalid_points_panicks() {
-        ProjectivePoint::<BLS12377Curve>::new([FEE::from(1), FEE::from(1), FEE::from(1)]);
+    fn create_invalid_points_panics() {
```

Comment:
We should evaluate if it still makes sense to have this test, given that it'll only panic only in debug idk if it is worth testing.